package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mapbox-upload-limits",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
I would move this version bump into a separate commit on master where the only change is `1.40` --> `1.4.1`. If you version the feature branches you could get into trouble as there is no clear definition of which is "the latest" version.

Reply:
ACK